partials/footer.hbs

```diff
@@ -2,28 +2,57 @@
 --}}
 
-<footer class="m-footer">
-  <div class="m-footer__content">
-    <p class="m-footer-copyright">
-      <span>{{@site.title}} &copy; {{date format="YYYY"}}</span>
-      <span>&nbsp; &bull; &nbsp;</span>
-      <span>{{t "Published with"}} <a href="https://ghost.org" target="_blank" rel="noopener">Ghost</a></span>
-    </p>
-
-    <nav class="m-footer-social">
-      {{#if @site.facebook}}
-        <a href="{{facebook_url @site.facebook}}" target="_blank" rel="noopener" aria-label="Facebook">
-          <span class="icon-facebook"></span>
-        </a>
-      {{/if}}
-      {{#if @site.twitter}}
-        <a href="{{twitter_url @site.twitter}}" target="_blank" rel="noopener" aria-label="Twitter">
-          <span class="icon-twitter"></span>
-        </a>
-      {{/if}}
+<footer class="footer">
+  <div class="footer-main">
+    <div class="footer-left">
+      <div>
+        <p class="footer-title">IEEE Student Branch</p>
+        <p class="footer-subtitle">University of California, Berkeley</p>
+      </div>
+      <nav class="footer-social">
+        {{#if @site.facebook}}
+          <a href="{{facebook_url @site.facebook}}" target="_blank" rel="noopener" aria-label="Facebook">
+            <span class="icon-facebook"></span>
+          </a>
+        {{/if}}
+        {{#if @site.twitter}}
+          <a href="{{twitter_url @site.twitter}}" target="_blank" rel="noopener" aria-label="Twitter">
+            <span class="icon-twitter"></span>
+          </a>
+        {{/if}}
+          <a href="https://www.linkedin.com/company/ieee-ucb/" target="_blank" rel="noopener" aria-label="LinkedIn">
+            <span class="icon-linkedin"></span>
+          </a>
+          <a href="https://www.instagram.com/ieeeucb/" target="_blank" rel="noopener" aria-label="Instagram">
+            <span class="icon-instagram"></span>
+          </a>
+      </nav>
+      <div class="footer-mailing-list">
+        <form action="https://eepurl.com/c-RpPr">
+          <button type="submit">Join our mailing list</button>
+        </form>
+      </div>
+    </div>
+    <nav class="footer-right">
+      <div class="footer-col">
+        <ul>
+          <li><a href="." rel="noopener">Home</a></li>
+          <li><a href="about" rel="noopener">About</a></li>
+          <li><a href="events" rel="noopener">Events</a></li>
+          <li><a href="apply" rel="noopener">Apply</a></li>
+        </ul>
+      </div>
+      <div class="footer-col">
+        <ul>
+          <li><a href="micromouse" rel="noopener">Micromouse</a></li>
+          <li><a href="hope" rel="noopener">HOPE</a></li>
+          <li><a href="industry" rel="noopener">Industrial Relations</a></li>
+        </ul>
+      </div>
     </nav>
-    <p class="m-footer-copyright jslicense">
-      <a href="{{asset "html/javascript.html"}}" rel="jslicense">{{t "JavaScript license information"}}</a>
-    </p>
+  </div>
+  <div class="footer-copyright">
+    <p>{{@site.title}} &copy; {{date format="YYYY"}}</p>
```

Comment:
I'm not sure if this is the correct copyright that we want to see. I think the site title is different from our actual copyright. `© Copyright 2020 IEEE Student Branch at UC Berkeley`